bindep.txt

```diff
@@ -0,0 +1,9 @@
+# This is a cross-platform list tracking distribution packages needed by tests;
```

Comment:
the other idea, is add docker here then us tools/test-setup.sh to ensure docker is started properly.

https://opendev.org/windmill/ansible-role-nodepool/src/commit/4666971ac5c2d2ac612aa085a002c1db60be09f6/tools/test-setup.sh